modules/nf-core/coptr/index/meta.yml

```diff
@@ -19,8 +19,7 @@ tools:
 input:
   - - meta:
         type: map
-        description: |
-          Groovy Map containing genome information
+        description: Name of the reference genome index
```

Comment:
Change this back, I meant the indexfasta description. Meta description is standardized.